### ISSUE TITLE
Support for Audio Description 

### ISSUE THREAD

**Issue by aptroot** [vidstack/player#874]
### Describe:

In order to comply with WCAG Level AA support is needed for Audio Description (AD). 

The intent of this Success Criterion is to provide people who are blind or visually impaired access to the visual information in a synchronized media presentation. The audio description augments the audio portion of the presentation with the information needed when the video portion is not available. During existing pauses in dialogue, audio description provides information about actions, characters, scene changes, and on-screen text that are important and are not described or spoken in the main sound track.

https://www.w3.org/WAI/WCAG21/Understanding/audio-description-prerecorded.html

### Anything Else?

Example of player with Audio Description (second audio track):
https://www.rijksoverheid.nl/documenten/videos/2023/03/16/de-duurzame-batterij-van-de-toekomst-wordt-in-nederland-ontwikkeld

**Comment by mihar-22** [comment]
Sorry for the late response here aptroot, this feature has been supported via HLS since the early days of our library. You will need to include the audio tracks with the appropriate labels in your HLS playlist and it should work as expected. If there's any additional feature requirements missing then let me know and we can reopen/discuss.
